install/02_machineautoscaler.crd.yaml

```diff
@@ -1,131 +1,139 @@
-              properties:
-                apiVersion:
-                  description: 'APIVersion defines the versioned schema of this representation
-                    of an object. Servers should convert recognized schemas to the
-                    latest internal value, and may reject unrecognized values. More
-                    info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#resources'
-                  type: string
-                kind:
-                  description: 'Kind is a string value representing the REST resource
-                    this object represents. Servers may infer this from the endpoint
-                    the client submits requests to. Cannot be updated. In CamelCase.
-                    More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#types-kinds'
-                  minLength: 1
-                  type: string
-                name:
-                  description: Name specifies a name of an object, e.g. worker-us-east-1a.
-                    Scalable resources are expected to exist under a single namespace.
-                  minLength: 1
-                  type: string
-              required:
-              - kind
-              - name
-              type: object
-          required:
-          - minReplicas
-          - maxReplicas
-          - scaleTargetRef
-          type: object
-        status:
-          description: Most recently observed status of a scalable resource
-          properties:
-            lastTargetRef:
-              description: LastTargetRef holds reference to the recently observed
-                scalable resource
-              properties:
-                apiVersion:
-                  description: 'APIVersion defines the versioned schema of this representation
-                    of an object. Servers should convert recognized schemas to the
-                    latest internal value, and may reject unrecognized values. More
-                    info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#resources'
-                  type: string
-                kind:
-                  description: 'Kind is a string value representing the REST resource
-                    this object represents. Servers may infer this from the endpoint
-                    the client submits requests to. Cannot be updated. In CamelCase.
-                    More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#types-kinds'
-                  minLength: 1
-                  type: string
-                name:
-                  description: Name specifies a name of an object, e.g. worker-us-east-1a.
-                    Scalable resources are expected to exist under a single namespace.
-                  minLength: 1
-                  type: string
-              required:
-              - kind
-              - name
-              type: object
-          type: object
+  scope: ""
```

Comment:
Non blocking nit: At some point we should add the resource here too, but since it's the default value I don't mind this being done later